apps/cli/commands/extract_reprocess.py

```
@@ -0,0 +1,54 @@
+    Example:
+        uv run apps/cli extract reprocess --since 7d
+
+    Raises:
+        typer.Exit: Exit with code 1 if reprocessing fails.
+    """
+    try:
+        # Parse since period
+        if not since.endswith("d"):
+            console.print(f"[red]✗ Invalid period format: {since}[/red]")
+            console.print("[yellow]Expected format: '7d', '30d', etc.[/yellow]")
+            raise typer.Exit(code=1)
```

Ruff (0.14.1)

Abstract `raise` to an inner function

(TRY301)